plugin.program.remote.control.browser/browse.py

```diff
@@ -17,31 +17,35 @@
 import threading
 
 
+logger = logging.getLogger('remotecontrolbrowser')
+logger.addHandler(logging.StreamHandler())
+logger.setLevel(logging.INFO)
+
+
 # If any of these packages are missing, the script will attempt to proceed
 # without those features.
 try:
     import alsaaudio
 except ImportError:
+    logger.debug('Missing Python package: alsaaudio')
     alsaaudio = None
-try:
-    import pulsectl
-except ImportError:
-    pulsectl = None
 try:
     import psutil
 except ImportError:
+    logger.debug('Missing Python package: psutil')
     psutil = None
+try:
+    import pulsectl
+except ImportError:
+    logger.debug('Missing Python package: pulsectl')
+    pulsectl = None
 try:
     import pylirc
 except ImportError:
+    logger.debug('Missing Python package: pylirc')
     pylirc = None
 
 
-logger = logging.getLogger('remotecontrolbrowser')
-logger.addHandler(logging.StreamHandler())
-logger.setLevel(logging.INFO)
-
-
 VOLUME_MIN = 0L
```

Comment:
The `L` literal for numbers is incompatible with python3 (https://www.python.org/dev/peps/pep-0237/). As the following variables are only used where the value is between `-1` and `101` it's not really necessary to define them as long. So it would be great if you can adjust the code.
Please tell me if you plan to do it in this PR or in the next one.

```suggestion
VOLUME_MIN = 0
```